tests/test_quantity.py

```diff
@@ -0,0 +1,398 @@
+import pytest
+
+import ansys.fluent.core.quantity as q
+
+
+def test_viscosity():
+    v = q.Quantity(1, "P")  # poise
+    conversion_output = v["kg m^-1 s^-1"]
+    assert conversion_output._real_value == 0.1
+
+
+def test_volume():
+    v = q.Quantity(1, "gal")
+    conversion_output = v["m^3"]
```

Comment:
Maybe it's just me but I'd prefer the API to be a bit more explicit `vm3 = v.convert("m^3")`

Reply:
Done.

Reply:
> Maybe it's just me but I'd prefer the API to be a bit more explicit `vm3 = v.convertTo("m^3")`. For me the `[]` syntax implies an array or dictionary element access.

@dnwillia-work agreed, we had unanimously agreed against dict style access here. We had decided to use the same naming as in pint which is just `to`. 

One important aspect of the naming is that whatever we choose must clearly indicate that the current object is unaffected and that a new object is returned. 

Your suggestion would need to be `convert_to`. @hpohekar, I see that it's gone in as `convertTo`.

Reply:
OK great.  If `pint` already provides a convention for what this method should be called I'd stick with that.

Reply:
Done. We have replaced ```converTo``` with ```to```.